### DISCUSSION TITLE
How to start downstream task of object detection on MAE ?

### DISCUSSION THREAD

**Discussion by wudizuixiaosa** [open-mmlab/mmselfsup#528] [How-to] [answered]
I would like to ask if I now support mae to do downstream testing, directly call benchmark？what else

**Comment by YuanLiuuuuuu** [comment]
Which kind of downstream task, classification, detection or segmentation ? Thanks!

**Comment by wudizuixiaosa** [comment]
detecion or segmentation is both ok，because I want use mask—rcnn，the dataset format they need seems similar

**Comment by wudizuixiaosa** [comment]
> Which kind of downstream task, classification, detection or segmentation ? Thanks!

hi bro，could you give me some help？

**Comment by YuanLiuuuuuu** [comment, marked as answer]
Currently, MMDet does not support ViT for Mask-RCNN, but I have already supported ViT for UpperNet in MMSeg. You can have a try, following this [guide](https://github.com/open-mmlab/mmsegmentation/tree/master/configs/mae). A kind note: There is no  need to convert the keys of the weights from MMSelfSup. Thanks!

**Comment by YuanLiuuuuuu** [comment]
If you have any other questions, please feel free to reopen it.

**Comment by austinmw** [comment]
@YuanLiuuuuuu Hi, does MAE currently support any downstream tasks in MMDet?
